frontend/src/patientApp/timeOfTest/PatientProfileContainer.tsx

```diff
@@ -1,154 +1,87 @@
-import moment from "moment";
-
-import { formatFullName } from "../../app/utils/user";
-import PatientForm from "../../app/patients/PatientForm";
-
-import {
-  RACE_VALUES,
-  ETHNICITY_VALUES,
-  GENDER_VALUES,
-} from "../../app/constants";
-
-import Button from "../../app/commonComponents/Button";
 import React, { useState } from "react";
 import { Redirect } from "react-router";
+import { useHistory } from "react-router-dom";
+import { connect, useSelector } from "react-redux";
 
-interface Props {
-  patient: any;
-}
+import Button from "../../app/commonComponents/Button";
+import PatientTimeOfTestContainer from "../PatientTimeOfTestContainer";
+import PatientProfile from "./PatientProfile";
 
-const PatientProfileContainer = (props: Props) => {
+const PatientProfileContainer = () => {
+  const history = useHistory();
   const [nextPage, setNextPage] = useState(false);
+  const [editPage, setEditPage] = useState(false);
+  const [prevPage, setPrevPage] = useState(false);
+  const patient = useSelector((state) => (state as any).patient as any);
 
-  const fullName = formatFullName(props.patient);
-  const race = RACE_VALUES.find((val) => val.value === props.patient.race)
-    ?.label;
-  const ethnicity = ETHNICITY_VALUES.find(
-    (val) => val.value === props.patient.ethnicity
-  )?.label;
-  const gender = GENDER_VALUES.find((val) => val.value === props.patient.gender)
-    ?.label;
-
-  const formattedAddress = () => {
-    const lastAddressLine = `${props.patient.city}${
-      props.patient.state && props.patient.city ? "," : ""
-    }${props.patient.city ? " " : ""}${props.patient.state}${
-      props.patient.state ? " " : ""
-    }${props.patient.zipCode}`;
+  history.listen((loc, action) => {
```

Comment:
couldn't figure out a way to make the back button work... except by doing this... 😅